### PR TITLE
44 plain default mobile nav

### PR DESCRIPTION
**Pull request by yuvilio**
Per discussion in #44, this PR switches the current header navbar nav with a basic starter out of the box mobile nav . It's aims are two fold:

1. Remove dependency on bootstrap navbar and it's setup. Utilizing navbar added more constrictions than benefits (shown below).

2. Give a starter option for a basic responsive nav, solving typical needs (mobile nav expand)

Before:
====

* The desktop menu did not show submenu elements
screenshot from 2017-06-13 13-14-12

* The mobile menu was always open (increasing usage of space) and it too did not show submenu (dropdown elements) 
 screenshot from 2017-06-13 13-14-39

After (with this PR):
====

* The desktop nav has a default hover state for dropdown submenu elements (courtesy of bootstrap hover dropdown): 

desktop-nav-demo

* In mobile, the menu starts hidden, and a bars hamburger appears on the top right to toggle it open. If the menu item has a dropdown, it's toggleable with a '+' link

mobile-nav-demo

As always, this is just an out of the box _default option_. Feel free to overwrite, if the design you're working with implies a different approach.

### CODE REVIEW

File: wp-content/themes/timber/assets/js/main.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var fitVidInit = function(){
 };
 
 var smoothScrollInit = function(){
-    $('a').smoothScroll();
+    $('a.scroll').smoothScroll();
```
Comment (ccorda) [inline wp-content/themes/timber/assets/js/main.js]: This feels unrelated (and I'd quibble on it). Can we revert and open as a separate issue?

File: wp-content/themes/timber/assets/js/main.js [hunk 2]
```diff
@@ -19,13 +19,36 @@ var quickShareInit = function(){
     quickShare();
 };
 
+var headerNavPosition = function(inMobileMq){
```
Comment (ccorda) [inline wp-content/themes/timber/assets/js/main.js]: This feels overly complex, using JS to repurpose markup when there is a simpler alternative -- using a `class="mobile-nav visible-xs"` for your mobile specific markup and menus.

**Comment by ccorda** [comment]
Left a few notes. 

One implementation note is that IMO based on common denominator in sites we build, hamburger nav should only be on mobile (xs). For sm-and-up, menu should be visible. For small, I'm ok if nav drops to its own line like this:

```
<div class="col-md-4 logo"></div>
<div class="col-md-8 nav"></div>
```
I'd push back on the JS to rearrange markup for mobile nav. Possible I'm misunderstanding the utility, but I don't think we need it. You can get collapsible nav without resorting to JS.

File: wp-content/themes/timber/assets/js/main.js (hunk 2 above)
Comment (yuvilio) [inline wp-content/themes/timber/assets/js/main.js]: I was trying to avoide redundancy of markup appearance with 4 lines for js rather than . It seemed cleaner in some ways but can go with redundancy on markup instead.

**Comment by yuvilio** [comment]
I think the js re-arrange is not a big deal, but it's agreed that it's not a must. Can go redudant nav markup (visible-xs, hidden-xs) route instead. 

Note that the hamburger menu is indeed only visible in mobile. In sm and up the menu is fully visibly (per screenshots above).

File: wp-content/themes/timber/assets/js/main.js (hunk 2 above)
Comment (yuvilio) [inline wp-content/themes/timber/assets/js/main.js]: Latest commit now removes this js re-arragnement in favor of bootstrap `visible-xs` approach instead.

**Comment by ccorda** [comment]
Looks good Yuval, :shipit:
